### DISCUSSION TITLE
will i get student pack benefits?

### DISCUSSION THREAD

**Discussion by StormAditya** [community/community#111087] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/83055431/45845a9e-3c0b-4f21-aefd-8148f4f76af0)
So i had gotten rejected 3 times and then 4th time even though i uploaded the same student ID I got approved but didn't get benefits yet so is it possible that they check again and reject it or is it confirmed now that i will get the benefits in some days ? 
![image](https://github.com/community/community/assets/83055431/b48684e0-12b2-48a9-b112-96e3d47cd3cd)

and one more thing is that it says You were last verified as a **_null_** on Mar 6, 2024, why is it null and not student ? what does null mean here ?

**Comment by samus-aran** [comment]
Hey @StormAditya  👋

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of the **Approved** verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

> and one more thing is that it says You were last verified as a null on Mar 6, 2024, why is it null and not student ? what does null mean here ?

There is a "null" value because you have not yet received your academic benefits but are approved. Once you do (usually 72 hours) that should update 👍

**Reply by StormAditya** [reply]
It's been 3 days now and well over 72 hours when will I be getting it?
